content/docs/1.6.0/references/helm-values.md

```diff
@@ -0,0 +1,232 @@
+| image.pullPolicy | string | `"IfNotPresent"` | `imagePullPolicy` to apply to all containers |
+
+### Service Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| service.manager.nodePort | string | `""` | NodePort port number (to set explicitly, choose port between 30000-32767) |
+| service.manager.type | string | `"ClusterIP"` | Define Longhorn manager service type. |
+| service.ui.nodePort | string | `nil` | NodePort port number (to set explicitly, choose port between 30000-32767) |
+| service.ui.type | string | `"ClusterIP"` | Define Longhorn UI service type. Available types: `ClusterIP`, `NodePort`, `LoadBalancer`, `Rancher-Proxy` |
+
+### StorageClass Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| persistence.backingImage.dataSourceParameters | string | `nil` | Specify the data source parameters for the backing image used in Longhorn StorageClass. This option accepts a json string of a map. e.g., `'{\"url\":\"https://backing-image-example.s3-region.amazonaws.com/test-backing-image\"}'`. |
+| persistence.backingImage.dataSourceType | string | `nil` | Specify the data source type for the backing image used in Longhorn StorageClass. If the backing image does not exists, Longhorn will use this field to create a backing image. Otherwise, Longhorn will use it to verify the selected backing image. |
+| persistence.backingImage.enable | bool | `false` | Set backing image for Longhorn StorageClass |
+| persistence.backingImage.expectedChecksum | string | `nil` | Specify the expected SHA512 checksum of the selected backing image in Longhorn StorageClass |
+| persistence.backingImage.name | string | `nil` | Specify a backing image that will be used by Longhorn volumes in Longhorn StorageClass. If not exists, the backing image data source type and backing image data source parameters should be specified so that Longhorn will create the backing image before using it |
+| persistence.defaultClass | bool | `true` | Set Longhorn StorageClass as default |
+| persistence.defaultClassReplicaCount | int | `3` | Set replica count for Longhorn StorageClass |
+| persistence.defaultDataLocality | string | `"disabled"` | Set data locality for Longhorn StorageClass (`disabled` or `best-effort`) |
+| persistence.defaultFsType | string | `"ext4"` | Set filesystem type for Longhorn StorageClass |
+| persistence.defaultMkfsParams | string | `""` | Set mkfs options for Longhorn StorageClass |
+| persistence.defaultNodeSelector.enable | bool | `false` | Enable Node selector for Longhorn StorageClass |
+| persistence.defaultNodeSelector.selector | string | `""` | We use NodeSelector when we want to bind PVC via StorageClass into desired mountpoint on the nodes tagged with its value |
+| persistence.migratable | bool | `false` | Set volume migratable for Longhorn StorageClass |
+| persistence.reclaimPolicy | string | `"Delete"` | Define reclaim policy (`Retain` or `Delete`) |
```

Comment:
The format of the enum values can be consistent. I see some of them are seperated by comma.

Reply:
Fixed